frontend/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+function Button(props) {

```

Comment:
Please use arrow functions. Also, destructure the props ahead. Apply to all.
```suggestion
const Button = ({ propsname }) => {

```